hatasmota/entity.py

```diff
@@ -98,27 +104,65 @@ def __init__(self, **kwds: Any):
         self._on_availability_callback: Callable[
             [bool], Coroutine[Any, Any, None]
         ] | None = None
+        self.uses_deep_sleep = False
+        self.deep_sleep_interval = None
+        self.last_up=0
```

Comment:
Please give this a better name, maybe `last_up_timestamp`, `timestamp_last_up` or something like that